docs/app/pages/getting-started/getting-started.component.less

```diff
@@ -0,0 +1,23 @@
+:host::ng-deep {
+    .nav-tabs {
+        margin-bottom: 15px;
+    }
+
+    .tab-content {
+        pre[class*="language-"] {
+            border-top: 1px solid #ddd;
+        }
+    }
+}
+
+h3 {
```

Comment:
Can you remove this? We should the standard heading margins, otherwise this looks different from our other pages.